index.html

```diff
@@ -573,27 +582,22 @@ <h2>
                   |paymentMethod|.{{PaymentMethodData/data}} into a string.
                   Rethrow any exceptions.
                   </li>
-                  <li>If |serializedData| is not null, and if required by the
-                  specification that defines the
-                  |paymentMethod|.{{PaymentMethodData/supportedMethods}}:
+                  <li>If |serializedData| is not null, and if the specification
+                  that defines the
+                  |paymentMethod|.{{PaymentMethodData/supportedMethods}}
+                  specifies <a>steps to convert and validate payment method
+                  data</a>:
                     <ol>
                       <li>Let |object| be the result of <a data-cite=
                       "ECMASCRIPT#sec-json.parse">JSON-parsing</a>
                       |serializedData|.
                       </li>
                       <li>
                         <p>
-                          Let |idl| be the result of [=converted to an IDL
-                          value|converting=] |object| to an IDL value of the
-                          type specified by the specification that defines the
-                          |paymentMethod|.{{PaymentMethodData/supportedMethods}}.
-                          Rethrow any exceptions.
-                        </p>
-                        <p>
-                          If required by the specification that defines the
-                          |paymentMethod|.{{PaymentMethodData/supportedMethods}},
-                          validate the members of |idl|. If a member's value is
-                          invalid, throw a {{TypeError}}.
+                          Run the <a>steps to convert and validate payment
+                          method data</a> from the specification that defines the
+                          |paymentMethod|.{{PaymentMethodData/supportedMethods}}
+                          on |object|. Rethrow any exceptions.
```

Comment:
Nit... Maybe rewritten as the following, to be a bit consistent with the stuff up on line 585: 
```suggestion
                          If provided by the specification that defines the
                          |paymentMethod|.{{PaymentMethodData/supportedMethods}}, 
                          run the <a>steps to convert and validate payment
                          method data</a> on |object|. Rethrow any exceptions.
```